doc/BEGINNERS_TUTORIAL.org

```diff
@@ -0,0 +1,184 @@
+and lines until finished. At least as important part of the work consists of editing the text:
+deleting and rewriting parts, moving sentences around or jumping to an earlier
+point to fix a discrepancy. 
+The crudest way to, for example, delete a certain line
+is moving the mouse to the line in question, clicking on the line and then
+deleting it by pressing backspace repeatedly. This is slow and
+inefficient, both because you have to take your hands from your keyboard and
+because repeatedly pressing backspace takes time. The more time you spend
+writing, the more time and energy is wasted. 
+To speed up editing, many editors use key chords for common editing tasks: control-c for copying and so on. 
+However, these types of shortcuts tend to have two problems. First, you have to press two keys at the
+same time, which is harder to coordinate and thus slower than pressing keys in a sequence.
+Second, you typically have to use your weakest fingers (pinkies) extensively and bend
+your wrists in unergonomic positions, which is uncomfortable for many and risks developing
+carpal tunnel syndrome in the long run. 
+By contrast, Spacemacs uses modal editing. Modal editing means that differend modes are used for editing and
+writing text. While this can sound complicated at first, in practice it can be
+learned quickly and once learned is unparallelled in speed and ergonomy. 
+Our earlier example of deleting a certain line of text (a very common edit
+task) can be achieved in Spacemacs by simply navigating to the line in question
+with the keys j and k (navigation keys) and pressing d (for "delete") two times!
+You might have noticed that this was achieved entirely without moving your fingers from
+your home row (the row where your fingers lie in rest when touch-typing) and
+without using modifier keys. 
+** Start the tutorial 
+The modal editing features of Spacemacs originate from a text editor called VI,
+and thus the modal editing tutorial is called eVIl tutor. Press SPC h T (that
+is, the spacebar followed by h and T) to familiarize yourself with 
+modal editig. 
+** Using the spacebar to launch commands
+Now that you are familiar with writing and editing text it is time to put the "Space"
+into Spacemacs. Because the spacebar is the most accessible key on the keyboard
+and is pressed by the strongest fingers (the thumbs), it is a natural choice for
+launching commands. You can think of it as the start menu of Spacemacs.
+** Buffers, windows and frames
+Because of its roots in Emacs, which was developed in the 80's
+before the advent of modern graphical user interfaces, Spacemacs has 
+a different name of what we normally call "windows": in Spacemacs these are
+called "frames". A frame is what pops up when you launch Spacemacs from your
+desktop shortcut. A frame contains windows and buffers.
+
+Windows in spacemacs are the visual spaces a frame is divided into. The default
```

Comment:
> Windows in spacemacs

First of all, 'Spacemacs' and not 'spacemacs' 😸 

Also, I think it's better to use 'Emacs' instead of 'Spacemacs' when you are talking about general behaviour or concepts that are not really related to Spacemacs. The main point is to avoid confusion about what Spacemacs is.